app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
```

Comment:
The import statement should use absolute imports instead of relative imports. Change `from app.cafe import Cafe` to `from cafe import Cafe` to comply with the checklist.